examples/purchase-tester/src/main/java/com/revenuecat/purchasetester/SubscriptionOptionExtensions.kt

```diff
@@ -4,7 +4,7 @@ import com.revenuecat.purchases.models.SubscriptionOption
 
 fun SubscriptionOption.toButtonString(isDefault: Boolean): String {
     val pricingPhasesString = pricingPhases.joinToString(separator = ",\n") { pricingPhase ->
-        "\t[${pricingPhase.formattedPrice}, ${pricingPhase.billingPeriod}, ${pricingPhase.billingCycleCount} cycles]"
+        "\t[${pricingPhase.price.formatted}, ${pricingPhase.billingPeriod}, ${pricingPhase.billingCycleCount} cycles]"
```

Comment:
i almost want this to be called `toString()`, though i know it's not really a function...

Reply:
oh i kinda like `localizedString`... `formatted` just feels to me like an adjective not a noun. but that could be a weird block in my head and no one else's haha